docs/component-inventory.md

```
@@ -570,24 +569,37 @@ def transform_fitted_pearson(
 4. **Z-Index** - Palmer Z-Index
 5. **scPDSI** - Self-calibrated Palmer
 
-#### Key Function
+#### Key Functions
```

markdownlint-cli2 (0.23.2)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)